addons/purchase_stock/views/product_views.xml

```diff
@@ -23,4 +23,21 @@
             </field>
         </field>
     </record>
+
+    <record id="product_view_kanban_catalog_purchase_only" model="ir.ui.view">
+        <field name="name">product.view.kanban.catalog.purchase_stock</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_view_kanban_catalog"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <div name="o_kanban_qty_available" position="after">
+                <div t-if="['consu', 'product'].includes(record.detailed_type.raw_value)"
```

Comment:
why show a (and "on hand") forecasted for consumable products?

Reply:
Good question :sweat_smile: 
I think it's just my mistake, I'll fix that

Reply:
In fact, it was like that before this PR, in `sale_stock`.
I'll assume it's an error and change that for only `product`, hoping somebody from the sale team will let use know if it's really needed.

cc @hote-odoo

Reply:
Hello :wave: 

It must have been an error. I don't know the flow of consumable products but if I'm not mistaken, they are not tracked.

FYI @hote-odoo doesn't work at Odoo anymore, you can ping me or the sales team directly for questions regarding the catalog :)